### PR TITLE
Update DNSType comments on compute.go

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/upgrade/compute.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func (u *Upgrade) CanUpgradeEtcd() bool {
 type ClusterState struct {
 	// KubeVersion describes the version of the Kubernetes API Server, Controller Manager, Scheduler and Proxy.
 	KubeVersion string
-	// DNSType
+	// DNSType the DNS add-on to be used.
```
Comment (SataQiu) [inline cmd/kubeadm/app/phases/upgrade/compute.go]: Thanks @xichengliudui 
But would that be a better description?
```suggestion
	// DNSType describes the type of DNS add-on used in the cluster
```

Comment (SataQiu) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: /cc @neolit123 
/cc @SataQiu

Comment (neolit123) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: yes, `DNSType describes the type of DNS add-on used in the cluster` is better.

File: cmd/kubeadm/app/phases/upgrade/compute.go [hunk 2]
```diff
@@ -60,7 +60,7 @@ func (u *Upgrade) CanUpgradeEtcd() bool {
...
 	DNSType kubeadmapi.DNSAddOnType
 	// DNSVersion describes the version of the kube-dns images used and manifest version
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/upgrade/compute.go]: please also change this to:
`DNSVersion describes the version of the DNS add-on`

Comment (xichengliudui) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: has been updated, `DNSVersion` is changed to `DNSType`.

Comment (neolit123) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: could you please also replace this:
> // DNSVersion describes the version of the kube-dns images used and manifest version

with:
> // DNSVersion describes the version of the DNS add-on.

Comment (xichengliudui) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: YES，updated.

**Review by neolit123 — COMMENTED** [review]
/lgtm
/approve

**Comment by neolit123** [comment]
@xichengliudui 
the comment:
```
// DNSType describes the type of DNS add-on used in the cluster. 
```
has trailing space at the end, that's why gofmt is failing.

File: cmd/kubeadm/app/phases/upgrade/compute.go [hunk 3]
```diff
@@ -60,9 +60,9 @@ func (u *Upgrade) CanUpgradeEtcd() bool {
 type ClusterState struct {
 	// KubeVersion describes the version of the Kubernetes API Server, Controller Manager, Scheduler and Proxy.
 	KubeVersion string
-	// DNSType
+	// DNSType describes the type of DNS add-on used in the cluster.
```
Comment (SataQiu) [inline cmd/kubeadm/app/phases/upgrade/compute.go]:
```suggestion
	// DNSType describes the type of DNS add-on used in the cluster.
```

Comment (SataQiu) [reply cmd/kubeadm/app/phases/upgrade/compute.go]: Small nits. Others lgtm.

**Comment by xichengliudui** [comment]
updated

**Comment by SataQiu** [comment]
/lgtm 
thanks

**Pull request merged**